Alexandr Antsiferov/lesson8/src/app/pages/Comment.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import CommentProfile from "../components/Comment";
+
+const Comment = props => {
+    const [comment, setComment] = useState({});
```

Comment:
```suggestion
    const [comment, setComment] = useState(null);
```
так как объект - всегда `truthy`, т.е. превращается в `true`